packages/cli/snippets/oauth2-pkce.js

```diff
@@ -14,6 +14,8 @@ const authentication = {
         state: '{{bundle.inputData.state}}',
         redirect_uri: '{{bundle.inputData.redirect_uri}}',
         response_type: 'code',
+        code_challenge_method: 'S256', // Added for PKCE
```

Comment:
The backend currently always return `S256`, but it's better for the client to not assume that.

```suggestion
        code_challenge_method: '{{bundle.inputData.code_challenge_method}}', // Added for PKCE
```